components/builder-depot-client/src/lib.rs

```diff
@@ -765,8 +760,8 @@ impl Client {
         res.read_to_string(&mut encoded)
             .map_err(Error::BadResponseBody)?;
         debug!("Body: {:?}", encoded);
-        let package: Package = serde_json::from_str::<Package>(&encoded)?;
-        Ok(package)
+        let package: json::Package = serde_json::from_str::<json::Package>(&encoded)?;
+        Ok(package.ident.into())
```

Comment:
Maybe `clone()` instead of `into()`? That'd be a little less ambiguous here, I think.

Reply:
We use `.into()` here so that we can return the core PackageIdent type which is getting used in other crates.

Reply:
Derp, of course.

It'll be so nice when there's only one kind of `PackageIdent` in the code 😄

Reply:
Yeah, that was to trigger the `From` impl to get our good 'ol `PackageIdent`. I think that (if not already) this conversion can be zero copy and zero allocations and most likely gets compiled down to a zero cost conversion at the end.